### ISSUE TITLE
how to set mask for those calculated anchors and what is the best cfg to use?

### ISSUE THREAD

**Issue by IbrahimBond** [AlexeyAB/darknet#7161]
i am training on small objects. i have recalculated the anchors for 6 clusters and width:288 height:512

the re calculated anchors are as follows: 13,  7,  18,  9,  27,  6,  23, 12,  36,  9,  53, 14
what is the correct way to set the masks?
can you check bellow if have set them 
cfg: 

> [net]
> # Testing
> #batch=1
> #subdivisions=1
> # Training
> batch=64
> subdivisions=32
> width=320
> height=544
> channels=3
> momentum=0.9
> decay=0.0005
> angle=0
> saturation = 1.5
> exposure = 1.5
> hue=.1
> 
> learning_rate=0.00261
> burn_in=1000
> max_batches = 500200
> policy=steps
> steps=400000,450000
> scales=.1,.1
> 
> [convolutional]
> batch_normalize=1
> filters=32
> size=3
> stride=2
> pad=1
> activation=leaky
> 
> [convolutional]
> batch_normalize=1
> filters=64
> size=3
> stride=2
> pad=1
> activation=leaky
> 
> [convolutional]
> batch_normalize=1
> filters=64
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [route]
> layers=-1
> groups=2
> group_id=1
> 
> [convolutional]
> batch_normalize=1
> filters=32
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [convolutional]
> batch_normalize=1
> filters=32
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [route]
> layers = -1,-2
> 
> [convolutional]
> batch_normalize=1
> filters=64
> size=1
> stride=1
> pad=1
> activation=leaky
> 
> [route]
> layers = -6,-1
> 
> [maxpool]
> size=2
> stride=2
> 
> [convolutional]
> batch_normalize=1
> filters=128
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [route]
> layers=-1
> groups=2
> group_id=1
> 
> [convolutional]
> batch_normalize=1
> filters=64
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [convolutional]
> batch_normalize=1
> filters=64
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [route]
> layers = -1,-2
> 
> [convolutional]
> batch_normalize=1
> filters=128
> size=1
> stride=1
> pad=1
> activation=leaky
> 
> [route]
> layers = -6,-1
> 
> [maxpool]
> size=2
> stride=2
> 
> [convolutional]
> batch_normalize=1
> filters=256
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [route]
> layers=-1
> groups=2
> group_id=1
> 
> [convolutional]
> batch_normalize=1
> filters=128
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [convolutional]
> batch_normalize=1
> filters=128
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [route]
> layers = -1,-2
> 
> [convolutional]
> batch_normalize=1
> filters=256
> size=1
> stride=1
> pad=1
> activation=leaky
> 
> [route]
> layers = -6,-1
> 
> [maxpool]
> size=2
> stride=2
> 
> [convolutional]
> batch_normalize=1
> filters=512
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> ##################################
> 
> [convolutional]
> batch_normalize=1
> filters=256
> size=1
> stride=1
> pad=1
> activation=leaky
> 
> [convolutional]
> batch_normalize=1
> filters=512
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [convolutional]
> size=1
> stride=1
> pad=1
> filters=18
> activation=linear
> 
> 
> 
> [yolo]
> mask = 3,4,5
> anchors = 13,  7,  18,  9,  27,  6,  23, 12,  36,  9,  53, 14
> classes=1
> num=6
> jitter=.3
> scale_x_y = 1.05
> cls_normalizer=1.0
> iou_normalizer=0.07
> iou_loss=ciou
> ignore_thresh = .7
> truth_thresh = 1
> random=0
> resize=1.5
> nms_kind=greedynms
> beta_nms=0.6
> 
> [route]
> layers = -4
> 
> [convolutional]
> batch_normalize=1
> filters=128
> size=1
> stride=1
> pad=1
> activation=leaky
> 
> [upsample]
> stride=2
> 
> [route]
> layers = -1, 23
> 
> [convolutional]
> batch_normalize=1
> filters=256
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [convolutional]
> size=1
> stride=1
> pad=1
> filters=18
> activation=linear
> 
> [yolo]
> mask = 1,2,3
> anchors = 13,  7,  18,  9,  27,  6,  23, 12,  36,  9,  53, 14
> classes=1
> num=6
> jitter=.3
> scale_x_y = 1.05
> cls_normalizer=1.0
> iou_normalizer=0.07
> iou_loss=ciou
> ignore_thresh = .7
> truth_thresh = 1
> random=0
> resize=1.5
> nms_kind=greedynms
> beta_nms=0.6
